android/app/src/main/java/host/exp/exponent/MainActivity.java

```diff
@@ -0,0 +1,54 @@
+package host.exp.exponent;
+
+import android.os.Bundle;
+
+import com.facebook.react.ReactPackage;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import expo.core.interfaces.Package;
+import host.exp.exponent.generated.DetachBuildConstants;
+import host.exp.exponent.experience.DetachActivity;
+
+public class MainActivity extends DetachActivity {
+
+  @Override
+  public String publishedUrl() {
+    return "exp://exp.host/@jmmchenry/Movement-Vitamins";
```

Comment:
we should start using something like we have for the web-app like @MovementVitaminTeam or something like that

Reply:
removed